src/main/java/io/github/townyadvanced/townyresources/listeners/TownyResourcesBukkitEventListener.java

```diff
@@ -44,15 +44,15 @@ public void onBlockBreak(BlockBreakEvent event) {
 	
 	@EventHandler(ignoreCancelled = true)
 	public void onBlockShearEntityEvent(BlockShearEntityEvent event) {
-		if(TownyResourcesSettings.isEnabled() && TownyResourcesSettings.areResourceExtractionLimitsEnabled()) {
+		if(TownyResourcesSettings.isEnabled() && TownyResourcesSettings.areResourceExtractionLimitsEnabled() && TownyResourcesSettings.areShearingExtractionLimitsEnabled()) {
 			//Dispensers cannot shear entities
 			event.setCancelled(true);
 		}	
 	}
 	
 	@EventHandler(ignoreCancelled = true)
 	public void onPlayerShearEntityEvent(PlayerShearEntityEvent event) {
-		if(TownyResourcesSettings.isEnabled()) {
+		if(TownyResourcesSettings.isEnabled() && TownyResourcesSettings.areShearingExtractionLimitsEnabled()) {
```

Comment:
This one should probably also have a `TownyResourcesSettings.areResourceExtractionLimitsEnabled()` added to it, I think its probably been missing it, no?

Reply:
You're correct. Added in 4934d53.